docs/modules/ROOT/pages/reference/glossary.adoc

```diff
@@ -0,0 +1,62 @@
+= Glossary
+
+== General
+
+=== Service
+A managed application that can be ordered, is auomatically provisioned and can be consumed by a service consumer.
+
+=== Service Consumer
+A customer-owned application that connects to a service as a client.
+
+=== Service Order
+The act of deploying a Kubernetes resource that triggers the provisioning of a new service instance.
+
+=== Service Instance
+An instance of a service offering.
+
+==== Cloud Instance
+An instance of cloud service offering by a cloud provider.
+
+==== Converged Service Instance
+An instance of a converged service blueprint.
+Either in the same namespace as the service order was made or in another namespace.
+
+== Service Location
+
+Services can be provisioned on various places:
+
+=== Converged Cluster
+Service runs on the same cluster as the service consumer.
+
+=== Service Cluster
+Service runs on one or more dedicated clusters, reserved for services.
+Usually deployed via a Helm Chart.
+
+=== Cloud Service
+A type of service architecture offered by a cloud provider such as AWS or GCP.
+
+=== Converged Service
+A type of service developed to run on a converged cluster.
+Usually deployed via a Helm Chart.
+
+==== Converged Service Architectures
+
+Standalone::
+A single node instance which is neither replicated nor clustered.
+
+Replicated::
+The service offers data replication to one or more other instances.
+
+Clustered::
+A fully clustered version of the service, usually offering active / active replication with fully automated cluster handling.
+
+== Personas
+
+=== Service User
+A human that interacts with {product}.
+
+=== Service Engineer
+A human that engineers and maintains a service (code).
+
+=== Support Engineer
```

Comment:
```suggestion
=== Service Operator
```

Reply:
Isn't that ambiguous with Operator from Kubernetes? I imagine that could lead to confusions.
What if we define "Service Engineer" as the one doing Ops and the "Service Maintainer" as the "programmer behind the scenes"?

Reply:
Oh yes, that could be misleading, right. I like the suggestion :+1: